tayoto_mikaellaantonette/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tayoto, Mikaella Antonette Page</title>
+    <link href="style.css" rel="stylesheet" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <div>
+      <div class="main-body">
+        <div class="body">
+          <div id="header">
+            <h1>TAYOTO, MIKAELLA ANTONETTE V.</h1>
+            <hr />
+          </div>
+          <div class="hobbies">
+            <h2>My Interests and Hobbies</h2>
+            <ul>
+              <li>Photography</li>
+              <li>Drawing</li>
+              <li>Online Gaming</li>
+              <li>Reading</li>
+              <li>Sleeping</li>
+            </ul>
+          </div>
+          <hr />
+          <div class="work">
+            <h2>My Works</h2>
+            <ol>
+              <li><a href="exercise1.html">Exercise 1</a></li>
+              <li><a href="exercise2.html">Exercise 2</a></li>
+              <li><a href="exercise3.html">Exercise 3</a></li>
+              <li><a href="exercise4.html">Exercise 4</a></li>
+              <li><a href="exercise5.html">Exercise 5</a></li>
```

Comment:
I noticed that your exercise1.html - exercise5.html is empty, you should replace your href with # so that the user will not be redirected to empty pages. Once you've made these changes, you can go ahead and delete exercise1.html through exercise5.html.

`<li><a href="#">Exercise 1</a></li>`